IPython/core/display.py

```diff
@@ -952,6 +972,10 @@ def _repr_jpeg_(self):
         if self.embed and (self.format == u'jpeg' or self.format == u'jpg'):
             return self._data_and_metadata()
 
+    def _repr_bmp_(self):
```

Comment:
We don't have anything that checks for `_repr_bmp_` at the moment, and I'm not convinced it's common enough to be worth adding code for. @minrk ?